### DISCUSSION TITLE
Housing and General

### DISCUSSION THREAD

**Discussion by abhiraj24** [snigi-gupta/AttendingUniversityAtBuffalo#27] [Q&A] [answered]
I have received an admit from MS in Data science in UB.
Happy to connect with you.
Actually I am pinging to know about certain aspects wrt to UB life and Buffalo, if you could spare some time:
1.)  What would be the initial expenditure breakdown that would be incurred as soon  as I land in USA.?
2.)   What would the monthly housing rent,grocery and utilities be like?
3.)   What are the first month charges wrt housing.. I am only are of security deposit.. is there any other charges incurred?

**Comment by snigi-gupta** [comment, marked as answer]
@abhiraj24 ,
1. initial expenditures is setup of your room/house, fees, groceries, phone & wifi plans, travel expenses. I carried about $3000 in cash and $2000 in my forex card.
2. Please check #12 and I am sure a lot of your questions will be answered there.
3. I paid first month, last month, and security deposit. Some landlords only take first month and security deposit. Depends.
4. There are many of your questions answered in the discussion section, I strongly advice you to browse through other people's questions too.
